src/OrchardCore/Orchard.Data/ServiceCollectionExtensions.cs

```diff
@@ -12,9 +12,13 @@
 using Npgsql;
 using Orchard.Data.Migration;
 using Orchard.Environment.Shell;
-using YesSql.Core.Indexes;
-using YesSql.Core.Services;
+using YesSql.Indexes;
 using YesSql.Storage.Sql;
+using YesSql;
+using YesSql.Provider.MySql;
+using YesSql.Provider.Sqlite;
+using YesSql.Provider.SqlServer;
+using YesSql.Provider.PostgreSql;
```

Comment:
YeslSql.Provider

All the extensions methods to add these providers should be at the root namespace. Not custom ones.